rca/settings/base.py

```diff
@@ -763,3 +763,7 @@
 
 pixel_limit = env.get("WAGTAILIMAGES_MAX_IMAGE_PIXELS")
 WAGTAILIMAGES_MAX_IMAGE_PIXELS = int(pixel_limit) if pixel_limit else 10000000
+
+ALLOW_EDITORIAL_PAGE_GENERATION = (
+    env.get("ALLOW_EDITORIAL_PAGE_GENERATION", "true").lower() == "true"
```

Comment:
This will default to `True`, would if be safer to default to `False`?

`env.get("ALLOW_EDITORIAL_PAGE_GENERATION", "false")`